tests/test_warm_reboot.py

```diff
@@ -966,6 +966,7 @@ def test_OrchagentWarmRestartReadyCheck(self, dvs, testlog):
         dvs.start_swss()
         time.sleep(5)
 
+    @pytest.mark.xfail(reason="Test unstable, blocking PR builds")
```

Comment:
We cannot revert tests as part of a feature PR. Please revert

Reply:
Reverted